### ISSUE TITLE
all-books error: JSONDecodeError when doing search

### ISSUE THREAD

**Issue by dmwyatt** [costis94/bookcut#13]
I'm getting error from other types of searches as well, but as an example here's the results of all-books author search.

```bash
$ bookcut all-books --author 'Stephen King'
```

results in:

```
 ____              _     ____      _
| __ )  ___   ___ | | __/ ___|   _| |_
|  _ \ / _ \ / _ \| |/ / |  | | | | __|
| |_) | (_) | (_) |   <| |__| |_| | |_
|____/ \___/ \___/|_|\_\\____\__,_|\__|


**********************************
Welcome to BookCut! I'm here to
help you to read your favourite books!
**********************************

Start searching for all books by Stephen king:
Connected to: http://www.openlibrary.org
Traceback (most recent call last):
  File "/home/dmwyatt/.local/bin/bookcut", line 8, in <module>
    sys.exit(entry())
  File "/home/dmwyatt/.local/pipx/venvs/bookcut/lib/python3.8/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/home/dmwyatt/.local/pipx/venvs/bookcut/lib/python3.8/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/home/dmwyatt/.local/pipx/venvs/bookcut/lib/python3.8/site-packages/click/core.py", line 1668, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/dmwyatt/.local/pipx/venvs/bookcut/lib/python3.8/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/dmwyatt/.local/pipx/venvs/bookcut/lib/python3.8/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/home/dmwyatt/.local/pipx/venvs/bookcut/lib/python3.8/site-packages/bookcut/bookcut.py", line 103, in bibliography
    lista = allbooks(author, ratio)
  File "/home/dmwyatt/.local/pipx/venvs/bookcut/lib/python3.8/site-packages/bookcut/bibliography.py", line 21, in main
    data = json.loads(jason)
  File "/usr/lib/python3.8/json/__init__.py", line 357, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.8/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.8/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 3 column 1 (char 2)
```

Not sure if it's an error from openlibrary or an error in parsing and don't have time to investigate at the moment...

**Comment by costis94** [comment]
Hi.

Actually I cannot reproduce the error, it seems its a parsing error.
Is the problem still occurring?

**Comment by dmwyatt** [comment]
My guess is openlibrary was having connectivity problems or something right when I was trying out bookcut for the first time, because it's working fine now.  Bad luck for me!

Maybe add a check to see if you're getting back valid JSON and provide a friendlier error message to the user?

**Comment by costis94** [comment]
Issue resolved in version 1.3.6.
